docs/conf.py

```diff
@@ -15,9 +15,10 @@
 import os
 import sys
 
+from j5 import __version__, __version_short__
```

Comment:
Does this break docs generation? Pretty sure it was after the `sys.path.insert()` so that python can be told what `j5` is

Reply:
I'm not even sure why this is in this PR...

I'll fix it